card/mod/01_core/set/all/tracked_attributes.rb

```diff
@@ -41,8 +41,9 @@ def extract_subcard_args! args={}
   if is_rule?
```

Comment:
this code, on the other hand, is still an ugly duckling. It's not your duckling, but it's still ugly ;)